THROTTLE_CONTROL_ROUTINES.s

```diff
@@ -131,8 +131,8 @@ DOIT		CA	PIF
 		CA	TIME1
 		TS	TTHROT
 
-# SINCE /AF/ IS NOT AN INSTANTANEOUS ACELERATION, BUT RATHER AN "AVERAGE" OF THE ACCELERATION LEVELS DURING
-# THE PRECEEDING PIPA INTERVAL, AND SINCE FP IS COMPUTED DIRECTLY FROM /AF/, FP IN ORDER TO CORRESPOND TO THE
+# SINCE /AF/ IS NOT AN INSTANTANEOUS ACCELERATION, BUT RATHER AN "AVERAGE" OF THE ACCELERATION LEVELS DURING
+# THE PRECEDING PIPA INTERVAL, AND SINCE FP IS COMPUTED DIRECTLY FROM /AF/, FP IN ORDER TO CORRESPOND TO THE
```

Comment:
`PRECEEDING` is used in [source](http://www.ibiblio.org/apollo/ScansForConversion/Luminary099/0795.jpg)